### ISSUE TITLE
TimeOut when setting multiple values in parallel (>20), retry expected

### ISSUE THREAD

**Issue by DutchmanNL** [ioBroker/ioBroker.hm-rpc#467]
**Is your feature request related to a problem? Please describe.**
As previously discussed in #340 we also discover issue when changing multiple actors in parallel.

Situation : 

- 28 Shutter actors of HomeMatic-IP
- All of them are controlled by ioBroker and using the Yahoo adapter for HomeKit integration

Controlling these stats by ioBroker runs fine, manually and by scripting up ~20 (as average) in parallel.
If I create a scene like in home kit or by blocky to control all shutter actors in parallel, I see at least 1-8 error messages showing the failed command with timeout like : 

```
hm-rpc.1 (5439) xmlrpc -> setValue ["00111D89949DBC:4","LEVEL",1] FLOAT
hm-rpc.1 (5439) Cannot call setValue: XML-RPC fault: Generic error (UNREACH)
```

I created a small script which writes a log entry every time channel 3 is acknowledge with a value. As you see in the summary of this log, I control all the shutters and a couple of them run fine but we have several time-outs with as result 3 of 25 did not respond
<img width="1334" alt="Schermafbeelding 2022-02-15 om 08 52 46" src="https://user-images.githubusercontent.com/7318445/154016905-12e19b99-0adf-4a29-9604-25e84cd48c9f.png">

**Describe the solution you'd like**
If the adapter has an active connection to the CCU, a retry should take place if an error occurs in communication

**Describe alternatives you've considered**
I created a script as kind of watchdog which monitors the states and verify's if the ACK value is set to true within 1,5 seconds after the state change.
This workarounds helps as failed state changes are matched by this script. Here I use the logic of maximum 3x retry before en final error occurs

**Additional context**

**Comment by DutchmanNL** [comment]
Update: as mentioned I try to handle this issue by a script which sets thee value again after 800ms if o acknowledge is provide by the adapter/CCU.

very interesting behaviour ! I see 4 out of 21 actors did not respond the first time, 3 needed 2 retry's and 1 even 3.
The error provided in initial issue did come 7 seconds after the successful acknowledge

so

1 - having a retry helps to handle time-out issues
2 - unclear error as the command is set (but could be related to the firs try of course ?
<img width="1040" alt="Schermafbeelding 2022-02-16 om 08 44 49" src="https://user-images.githubusercontent.com/7318445/154218888-1ccf4d53-4ddc-4b37-b55e-ddfb7799e2a7.png">

**Comment by Apollon77** [comment]
I have a comparable issue:

```
2022-06-19 19:54:02.960  - ESC[31merrorESC[39m: hm-rpc.0 (25917) xmlrpc -> setValue ["OEQ0208665:4","STATE",true] BOOL
2022-06-19 19:54:02.965  - ESC[31merrorESC[39m: hm-rpc.0 (25917) Cannot call setValue: XML-RPC fault: Failure
```

Could be because of LAN-GW communication issues or such, but never found out whats up

**Comment by foxriver76** [comment]
Please try GitHub version

**Comment by DutchmanNL** [comment]
> Please try GitHub version

thank you! Will need some time to reproduce and provide feedback

**Comment by reutli** [comment]
I'm facing the same issue, but not with such much shutter actors ...
I'm trying to switch my programs from CCU to ioBroker and now running in this issue.
`
2022-07-15 21:10:01.156  - error: hm-rpc.0 (13393) xmlrpc -> setValue ["OEQ0290772:1","LEVEL",1] FLOAT
2022-07-15 21:10:01.156  - error: hm-rpc.0 (13393) Cannot call setValue: XML-RPC fault: Failure
2022-07-15 21:15:01.281  - error: hm-rpc.1 (25403) xmlrpc -> setValue ["001397099C4727:4","LEVEL",1] FLOAT
2022-07-15 21:15:01.281  - error: hm-rpc.1 (25403) Cannot call setValue: XML-RPC fault: Generic error (UNREACH)
`
also discussing this in the forum [https://forum.iobroker.net/topic/56331/hm-rpc-xml-rpc-fehler-bei-jalousien-aktoren/8](url)

As I'm already using the latest version (1.15.12) I'm not sure if my issue is the same?

**Comment by Apollon77** [comment]
UNREACH normally means that the CCU can not reach the device ... does control works via CCU webui?

**Comment by reutli** [comment]
sure I know and yes I can trigger the by CCU WebUI.
I'm facing issues regarding the "setValue" failure.

**Comment by EuloTV** [comment]
i have a similar if not the same problem:

hm-rpc.4 | 2023-10-17 13:44:40.127 | error | Cannot call setValue: XML-RPC fault: Generic error
hm-rpc.4 | 2023-10-17 13:44:40.126 | error | xmlrpc -> setValue ["000A18A9A674CB:1","SET_POINT_TEMPERATURE",21] FLOAT

**Comment by EuloTV** [comment]
by trigger over the WebUI of the CCU have i no problem...

> i have a similar if not the same problem:
> 
> hm-rpc.4 | 2023-10-17 13:44:40.127 | error | Cannot call setValue: XML-RPC fault: Generic error hm-rpc.4 | 2023-10-17 13:44:40.126 | error | xmlrpc -> setValue ["000A18A9A674CB:1","SET_POINT_TEMPERATURE",21] FLOAT
> 
> EDIT:
> 
> hm-rpc.4 are my homematic IP devices that are registered in the CCU.
> 
> Adapter version 1.15.19

by trigger over the WebUI of the CCU have i no problem...

**Comment by DutchmanNL** [comment]
@EuloTV as this issue is closed and also I as starter can not reopen it, please create a new issue and reference to this one.
closed issue are not that visible for developers, also I have a kind of filter in my notifications otherwise I get overloaded by git
